hazelcast/src/test/java/com/hazelcast/internal/metrics/metricsets/RuntimeMetricSetTest.java

```diff
@@ -106,7 +107,7 @@ public void run() throws Exception {
     @Test
     public void availableProcessors() {
         LongGauge gauge = metricsRegistry.newLongGauge("runtime.availableProcessors");
-        assertEquals(runtime.availableProcessors(), gauge.read());
+        assertEquals(RuntimeAvailableProcessors.get(), gauge.read());
```

Comment:
I do not agree. This is the RuntimeMetricSet, so it will report what is in the Runtime. Not some kind of mocked value.